pages/about-us/index.tsx

```diff
@@ -0,0 +1,90 @@
+import NavBar from '../../lib/components/navbar';
+import navLinks from '../../lib/components/navbar/links';
+import Header from '../../lib/components/header';
+import { Container, Grid, Text, UnorderedList, ListItem } from '@chakra-ui/react';
+import CommitteeGroupProfile from '../../lib/components/committee-group-profile';
+import LearnMore from '../../lib/components/learn-more';
+
+const AboutUs: React.FC = (props) => {
+  return (
+    <>
+      <NavBar navLinks={navLinks} />
+      <Grid gap={'150px'}>
```

Comment:
I'm not a huge fan of Grid, have you tried using the Flex component to do this?
https://chakra-ui.com/docs/components/layout/flex
This should make your components more responsive as well, use width and maxWidth to your advantage.